packages/browser/src/plugins/ajs-destination/loader.ts

```diff
@@ -29,14 +33,37 @@ function recordLoadMetrics(fullPath: string, ctx: Context, name: string): void {
   }
 }
 
+export function buildIntegration(
+  integrationSource: LegacyIntegrationSource,
+  integrationSettings: { [key: string]: any },
+  analyticsInstance: Analytics
+): LegacyIntegration {
+  let integrationCtr: LegacyIntegrationBuilder
+  // GA and Appcues use a different interface to instantiating integrations
+  if ('Integration' in integrationSource) {
+    const analyticsStub = {
+      user: (): User => analyticsInstance.user(),
+      addIntegration: (): void => {},
+    }
+
+    integrationSource(analyticsStub)
+    integrationCtr = integrationSource.Integration
+  } else {
+    integrationCtr = integrationSource
+  }
+
+  const integration = new integrationCtr(integrationSettings)
+  integration.analytics = analyticsInstance
+
+  return integration
+}
+
 export async function loadIntegration(
```

Comment:
This is a good opportunity  to rename "loadIntegration" to something like "loadRemoteDestination" -- since now we have multiple ways to load destinations. The word "build" vs "load" can have overlapping meanings.